internal/cuckoo/cuckoo.go

```diff
@@ -33,6 +35,18 @@ type CuckooHasher struct {
 
 // NewCuckooHasher instantiates a CuckooHasher struct.
 func NewCuckooHasher(size uint64, seeds [Nhash][]byte) *CuckooHasher {
+	// get randombyte from crypto/rand
+	var rb [8]byte
+	if _, err := crand.Read(rb[:]); err != nil {
+		panic(err)
+	}
+
+	// WARNING: math/rand is not concurrency-safe
+	// replace with crypto/rand if that's what you want
+
+	// seed math/rand with crypto/rand
+	rand.Seed(int64(binary.LittleEndian.Uint64(rb[:])))
+
```

Comment:
move this to `NewCuckoo`, as the randomness is only needed for those that insert into the cuckoo hash table.

Reply:
good catch. Fixed it.